Dalamud/Game/ChatHandlers.cs

```diff
@@ -271,17 +286,22 @@ private void AutoUpdatePlugins()
         var notifications = Service<NotificationManager>.GetNullable();
 
         if (chatGui == null || pluginManager == null || notifications == null)
+        {
+            Log.Warning("Aborting auto-update because a required service was not loaded.");
             return;
+        }
 
         if (!pluginManager.ReposReady || !pluginManager.InstalledPlugins.Any() || !pluginManager.AvailablePlugins.Any())
```

Comment:
We should retry immediately when this happens, once we saw the first notice.

Reply:
How would you like me to implement this?

Any future `Notice` will already attempt to re-trigger an auto update (as the `startedAutoUpdaingPlugins` bool hasn't been tripped yet). If you want the first Notice to still perform a retrigger I can make this method return a bool and retry from the caller, but that feels like we're just racing code at that point.